docs/index.html

```diff
@@ -728,7 +729,7 @@ <h2 id="Transforms">Transforms</h2>
           <div id=append class="doc-item">
             <div class="doc-heading">
               <ul class="links">
-                <li><a href="https://github.com/caolan/highland/blob/3.0.0-beta.9/lib/index.js#L3844"><i class="fi-pencil"></i> Source</a></li>
+                <li><a href="https://github.com/caolan/highland/blob/3.0.0-beta.9/lib/index.js#L4010"><i class="fi-pencil"></i> Source</a></li>
```

Comment:
I started updating these line references after the new method bumped some lines down, but then noticed that there were actually more than just methods under the newly added one with incorrect line references so I was unsure if you guys wanted me to update these docs or not. I can go and finish the rest or undo the changes if need be.